### DISCUSSION TITLE
Enjoy Macbook pro 系统14.5无法加载本地音频资源

### DISCUSSION THREAD

**Discussion by xiaochuanyu1978** [ZuodaoTech/everyone-can-use-english#809] [Q&A] [answered]
Macbook pro 电脑系统14.5 (23F79)，芯片 M3 MAX,无法加载本地音频资源，Enjoy版本 Enjoy-0.3.0-arm64和Enjoy-0.2.13-arm64都无法加载本地资源，下载资源会报错，加载本地资源无法选择，如下图所示。
文件下载地址：https://hf-mirror.com/ggerganov/whisper.cpp/blob/main/ggml-tiny.en.bin，

![下载报警](https://github.com/user-attachments/assets/e21d72e1-3873-4406-a850-5a5d1873f7ef)
![下载文件](https://github.com/user-attachments/assets/14492b49-f84f-4ad3-8e07-e0254b6c91ae)
![Enjoy 版本](https://github.com/user-attachments/assets/68e687d8-1b50-40b6-a39d-4617e16d7934)
![WX20240715-220857@2x](https://github.com/user-attachments/assets/a7f8afba-99f3-42e7-822c-2c37536f1e06)
![WX20240715-221029@2x](https://github.com/user-attachments/assets/fc2118ae-8fc2-408b-bd8e-0c15d5b2471b)

![操作说明](https://github.com/user-attachments/assets/d853622d-31d0-4dc7-a2c7-a51ebf8fac4c)
![电脑](https://github.com/user-attachments/assets/f9f9738b-ed34-40a1-acd9-5f4819218337)
![电脑版本](https://github.com/user-attachments/assets/5679e9ea-1e3f-4e01-9556-4200c32a79ec)

**Comment by an-lee** [comment]
这里添加资源是添加音频或者视频文件，不是 模型文件。

你为啥要添加模型文件？

**Comment by xiaochuanyu1978** [comment, marked as answer]
谢谢，我知道如何使用了，在设置里面选择本地，并且选择对应下载模型或者点击下载就行


![设置](https://github.com/user-attachments/assets/2ab9738f-2a5d-43c0-9704-c0b354332d70)
![模型](https://github.com/user-attachments/assets/8b0f6f6a-a6d9-480d-944a-552c889287c4)
